Mage.Sets/src/mage/cards/g/GiantOx.java

```diff
@@ -0,0 +1,37 @@
+package mage.cards.g;
+
+import mage.MageInt;
+import mage.abilities.keyword.CrewWithToughnessAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author varaghar
+ */
+public class GiantOx extends CardImpl {
+
+    public GiantOx(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.CREATURE},"{1}{W}");
+        this.subtype.add(SubType.OX);
+
+        this.power = new MageInt(0);
+        this.toughness = new MageInt(6);
+
+        // Giant Ox crews Vehicles using its toughness rather than its power.
+        this.addAbility(new CrewWithToughnessAbility());
```

Comment:
this should use the singleton instance rather than the constructor, which shouldn't be accessible